.github/workflows/spellcheck.yml

```diff
@@ -416,6 +431,7 @@ jobs:
 #W:Ok
 #W:OpenBSD
 #W:openldap
+#W:OpenLDAP
 #W:Opensocket
```

Comment:
The capital letter here doesn't seem right.

Reply:
Which one? You mean `OpenLDAP` is not right?

Reply:
No; I complained about GitHub not letting me put my comments on the right lines. In most cases, my comment refers to the line immediately above (Opensocket in this case), but if Github didn't let me do that, then I mention the actual word in my comment. Maybe there's a better way to do this kind of review.

Reply:
nothing uses Opensocket, removed